### PR TITLE
Docker_Image.rst: Mention no support for Python 2

### PR DESCRIPTION
**Pull request by ayush1999**
Added the line:"Docker does not support python 2
code analysis."

Closes https://github.com/coala/documentation/issues/477

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by palash25** [comment]
ack 6674163

**Comment by ayush1999** [comment]
@Makman2 @jayvdb Any more changes? :)

**Comment by Makman2** [comment]
1. Quoting the newcomers guide:
> Do not mention users unrelated to a particular issue/question. This also means you should not mention the person that was last online. For example, mentioning someone on a Github issue will subscribe them to that issue, even if they don’t want to be a part of the discussion. However, mentioning someone is okay if they really need to see your message.

Your PR will be reviewed, no need to explicitly ping maintainers for the reason described in the quote.

2. Please look at your commit message, it's not imperative tense + the body is redundant (the described change is that what the diff explains itself) + you could be a bit more precise in the shortlog, adding docs in a docs-file is not really revealing.

**Comment by Makman2** [comment]
unack 6674163

**Comment by ayush1999** [comment]
Oh. Sorry for the trouble, won't happen again. I hope I have made the required changes though.

**Comment by newbazz** [comment]
@ayush1999 the body of commit could be a bit better, otherwise you have made the required changes.

**Comment by Makman2** [comment]
Hm shortlog has no real verb... maybe `Mention no support for Python 2`?

**Comment by ayush1999** [comment]
Done!

File: Users/Docker_Image.rst [hunk 1]
```diff
@@ -16,6 +16,7 @@ system, Docker allows applications to use the same kernel as the system
 that they're running on and only requires applications being shipped
 with things not already running on the host computer. This gives a
 significant performance boost and reduces the size of the application.
+Also, Docker does not support Python 2 analysis.
```
Comment (Makman2) [inline Users/Docker_Image.rst]: I think this belongs to "coala as a docker image". Docker itself supports every kind of software more or less ;)

Comment (Makman2) [reply Users/Docker_Image.rst]: You could put this into a note, like
```
.. note::

The coala Docker image does not support Python 2 analysis.
```
Or so

**Comment by Makman2** [comment]
Commit message: Make the "No" lowercase -> "no" ;D

**Comment by Makman2** [comment]
unack d978fc9

**Comment by ayush1999** [comment]
Done! Any other changes? :)

**Comment by shreyans800755** [comment]
@ayush1999 
No need to mention in the commit body that this note was not present earlier. That is implicit. Please remove that.
unack edff7d7

**Comment by ayush1999** [comment]
@shreyans800755 Thanks for pointing out! I removed that part. :)

**Comment by shreyans800755** [comment]
`Added this information` -> `Added this information in documentation.`

**Comment by shreyans800755** [comment]
ack cdffcd6

File: Users/Docker_Image.rst [hunk 2]
```diff
@@ -17,6 +17,10 @@ that they're running on and only requires applications being shipped
 with things not already running on the host computer. This gives a
 significant performance boost and reduces the size of the application.
 
+.. note::
+
+   The Coala Docker image does not support Python 2 analysis.
```
Comment (Makman2) [inline Users/Docker_Image.rst]: > coala is always written with a lower case c.

**Comment by Makman2** [comment]
unack cdffcd6

File: Users/Docker_Image.rst (hunk 2 above)
Comment (Makman2) [reply Users/Docker_Image.rst]: Also please use the same indent everywhere, I think it's 2 spaces (please recheck that)

**Comment by ayush1999** [comment]
@Makman2 Pl check. Thanks :)

File: Users/Docker_Image.rst (hunk 2 above)
Comment (newbazz) [reply Users/Docker_Image.rst]: Yeah its of two spaces.

**Comment by newbazz** [comment]
@ayush1999 cange the commit body such that the `coala` starts with a samll `c`.

**Comment by ayush1999** [comment]
@newbazz Done!

File: Users/Docker_Image.rst [hunk 3]
```diff
@@ -17,6 +17,10 @@ that they're running on and only requires applications being shipped
 with things not already running on the host computer. This gives a
 significant performance boost and reduces the size of the application.
 
+  .. note::
```
[the remark this answers is not in the document]
Comment (newbazz) [reply Users/Docker_Image.rst]: look at https://github.com/getpelican/pelican/issues/298 and solve this.

**Comment by shreyans800755** [comment]
ack 34ebfeb

File: Users/Docker_Image.rst (hunk 3 above)
Comment (Makman2) [inline Users/Docker_Image.rst]: Unnecessary indent of `.. note::` by 2 spaces.

File: Users/Docker_Image.rst [hunk 4]
```diff
@@ -17,6 +17,10 @@ that they're running on and only requires applications being shipped
...
+
+    The coala Docker image does not support Python 2 analysis.
```
Comment (Makman2) [inline Users/Docker_Image.rst]: Please move it to `coala as a Docker Image`, this has more relevance in that paragraph^^

**Comment by Makman2** [comment]
unack 34ebfeb

**Comment by ayush1999** [comment]
@Makman2 I hope I've made all the required changes. :)

**Comment by ayush1999** [comment]
@newbazz Done.

**Comment by newbazz** [comment]
LGTM

File: Users/Docker_Image.rst [hunk 5]
```diff
@@ -58,6 +58,10 @@ on your code in the working directory, like this:
 .. seealso::
   See also https://hub.docker.com/r/coala/base/.
 
+.. note::
+
+  The coala Docker Image does not support Python 2 analysis.
```
Comment (Makman2) [inline Users/Docker_Image.rst]: why is image's first letter upper-case?

**Comment by Makman2** [comment]
I would rather recommend to write it lowercase, because it's not referring to a name or title. Do it then for both, inside the commit body and in the .rst

**Comment by Makman2** [comment]
unack a3c1e9315af9b2ee57230b9204da9b51273bd82f

**Comment by ayush1999** [comment]
@Makman2 Made the required changes. Pl. check.

**Comment by jayvdb** [comment]
Please remove "This commit adds this infor..." in the commit body , and rebase your commit.

**Comment by ayush1999** [comment]
@jayvdb Done :) Please check. :+1:

**Comment by jayvdb** [comment]
ack d7f0bcffee5404de

**Comment by jayvdb** [comment]
@rultor merge

**Pull request merged**